activesupport/lib/active_support/actionable_error.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "rails/command"
```

Comment:
Active Support can't depend on Railties. We should register the command as the action to run on that exception in a Railtie if we want to use Rails Commands for them.